src/cmd/kubectl.rs

```diff
@@ -514,9 +514,7 @@ where
             ),
         },
         Err(e) => warn!(
-            "Unable to execute describe on secrets: {}. it may not exist anymore?",
```

Comment:
Why did you remove this? Wasn't it useful?

Reply:
nop, IMO